components/playback/controls/player/AddCustomer.vue

```diff
@@ -6,16 +6,54 @@
     </span>
     <div id="uuid" style="top: 40px; left: 50px; position: absolute">
       <InputWidget
-        :title_label="label_uuid"
-        :placeholder="keyplaceholder_encode"
-        :spellcheck="spellchecking"
+        :title_label="'Enter Alphanumeric ID'"
+        :placeholder="'2VSckkBYr2An3dqHEyfRRE'"
+        :spellcheck="false"
         :invalid_text="error_text_uuid"
-        :input_width="entry_width"
-        :disabled="disallow_entry"
+        :input_width="400"
+        :disabled="false"
```

Comment:
the default value for this prop is already `false`, so it's not necessary to specify that here

Reply:
Agreed.